libr/anal/var.c

```diff
@@ -750,7 +750,7 @@ R_API void r_anal_extract_rarg(RAnal *anal, RAnalOp *op, RAnalFunction *fcn, int
 				r_meta_set_string (anal, R_META_TYPE_VARTYPE, op->addr, vname);
 				free (name);
 				free (type);
-				*count++;
+				count++;
```

Comment:
thats wrong

Reply:
What is the point of the * in `*count++`?

Reply:
I guess it should be `(*count)++`?